### PR TITLE
Clean up PEG.js package references

### PR DESCRIPTION
**Pull request by swissspidy**
#1152 duplicated the pegjs reference in `package.json` and also added `phpegjs` as a regular dependency instead of a dev dependency.

This PR attempts to fix that.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,8 +22,7 @@
     "lodash": "^4.17.4",
     "moment": "^2.18.1",
     "moment-timezone": "^0.5.13",
-    "pegjs": "0.10.0",
```
Comment (youknowriad) [inline package.json]: Do we really need it in dependencies or should it be moved to `devDependencies`? I guess it's used only for generating parsers in dev, am I wrong?

Comment (swissspidy) [inline package.json]: Right. I misread that part in #1152. It only just a dev dependency before.

**Review by youknowriad — APPROVED** [review]
👍

**Comment by nylen** [comment]
Thanks for fixing this!

**Pull request merged**